<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output</class>
 <widget class="QWidget" name="output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>显示</string>
  </property>
  <widget class="QPushButton" name="pushButton_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>按姓名输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_id">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>600</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>按编号输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_grade">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>600</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>按职业输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>150</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_grade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>150</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>职业</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>150</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_salary">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>150</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>月薪</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
